src/containers/quiz/question-answer/Answer.styles.ts

```diff
@@ -0,0 +1,28 @@
+type AnswerState = 'correct' | 'incorrect'
```

Comment:
Maybe move it to global types folder? What do you think?

Reply:
I added AnswerStatus type inside of `Answer.types.ts` since for now it is used only in `Quiz`folder